src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightLambda.cs

```diff
@@ -12,6 +13,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+    [DebuggerDisplay("{DebugView,nq}")]
```

Comment:
@VSadov, do you know what the intention of the DebugView property was?  Such a DebuggerDisplay was my assumption, but other than a test using reflection to access the property, I couldn't find actual usage of it.

Reply:
I always thought this is for debugging. - When you hover over a lambda, you may see the tree for its body, which would be very helpful. Whether it actually works nowdays - I am not sure. 
There were problems in the past with visualization not lighting up because right attibutes were not applied or the whole thing was trimmed away.

Reply:
I just experimented with this.  There's only place this type is constructed, it's never handed back to user code, and it's immediately converted to a delegate, which means this debug display isn't useful for consumers, only for the developer of the Expressions library itself.  As such, I'm going to make this DEBUG-only.

Reply:
Ah, actually, I take that back... the created delegate's Target is the LightLambda, so there's some value in it.  Given that, I'm going to keep the DebuggerDisplay.